ibis/tests/expr/test_operations.py

```diff
@@ -162,9 +163,11 @@ class Aliased(Base):
         name: str
 
     ketto = Aliased(one, "ketto")
-    subs = {Name("one"): Name("zero"), two: ketto}
 
-    new_values = values.replace(subs)
+    first_rule = EqualTo(Name("one")) >> Name("zero")
+    second_rule = EqualTo(two) >> ketto
+
+    new_values = values.replace(first_rule | second_rule)
```

Comment:
What are the semantics of the `|` operator here?

Reply:
The `|` operator constructs an `AnyOf(first_rule, second_rule)` pattern.